android/src/app/organicmaps/MwmActivity.java

```diff
@@ -1389,10 +1391,12 @@ private void showMainMenu(boolean show)
   }
 
   @Override
-  @SuppressWarnings("deprecation") // https://github.com/organicmaps/organicmaps/issues/3631
   public void onRoutingPlanStartAnimate(boolean show)
   {
-    int offset = mCurrentWindowInsets.getSystemWindowInsetTop();
+    if (mCurrentWindowInsets == null) {
+      return;
```

Comment:
Can we 1) crash here in debug; 2) put here detailed TODO that it's a workaround?

Reply:
I could not reproduce the crash, so yeah I'll add a comment so we know this section may sometimes be called before the insets are initialized